org.lflang/src/org/lflang/generator/cpp/CppGenerator.kt

```diff
@@ -28,13 +28,10 @@ package org.lflang.generator.cpp
 
 import org.eclipse.emf.ecore.resource.Resource
 import org.eclipse.xtext.generator.IFileSystemAccess2
-import org.eclipse.xtext.generator.IGeneratorContext
 import org.lflang.*
 import org.lflang.TargetConfig.Mode
 import org.lflang.Target
-import org.lflang.generator.CodeMap
-import org.lflang.generator.GeneratorBase
-import org.lflang.generator.TargetTypes
+import org.lflang.generator.*
```

Comment:
I think most coding guidelines suggest to have specific imports without wildcards.

Reply:
That makes sense. I'll configure IntelliJ so that it doesn't do that.